ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml

```diff
@@ -619,6 +619,9 @@ tests:
   cluster: build05
   cron: 21 22 */1 * 1-5
   steps:
+    env:
+      E2E_TESTS_CONFIG: |
```

Comment:
Why just not to use `CNF_BRANCH: release-4.11` ?

Reply:
than you need to change the env config to CNF_BRANCH [here](https://github.com/openshift/release/pull/32086/files#diff-ffd8a5dab2ba9b8eb035e5a4dd1ed4db6f043c718d3a04cca08a458d10fca674R13) and i was thinking to make it more generic so we can add more env variables in the future.

Reply:
ok, fair enough